src/views/TechnicalAssistance/TrbAdminTeamHome.tsx

```diff
@@ -0,0 +1,648 @@
+import React, { useMemo, useState } from 'react';
+import { CSVLink } from 'react-csv';
+import { useTranslation } from 'react-i18next';
+import {
+  CellProps,
+  Column,
+  useFilters,
+  useGlobalFilter,
+  usePagination,
+  useSortBy,
+  useTable
+} from 'react-table';
+import { useQuery } from '@apollo/client';
+import {
+  Button,
+  GridContainer,
+  IconFileDownload,
+  Table
+} from '@trussworks/react-uswds';
+import classnames from 'classnames';
+import i18next from 'i18next';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import PageHeading from 'components/PageHeading';
+import PageLoading from 'components/PageLoading';
+import GlobalClientFilter from 'components/TableFilter';
+import TablePageSize from 'components/TablePageSize';
+import TablePagination from 'components/TablePagination';
+import GetTrbAdminTeamHomeQuery from 'queries/GetTrbAdminTeamHomeQuery';
+import { GetTrbAdminTeamHome } from 'queries/types/GetTrbAdminTeamHome';
+import { TrbAdminTeamHomeRequest } from 'types/technicalAssistance';
+import { formatDateLocal } from 'utils/date';
+import globalFilterCellText from 'utils/globalFilterCellText';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon
+} from 'utils/tableSort';
+
+function getPersonVal(name: string, component?: any) {
+  return `${name}${typeof component === 'string' ? `, ${component}` : ''}`;
+}
+
+export const trbRequestsCsvHeader = [
+  i18next.t<string>('technicalAssistance:table.header.submissionDate'),
+  i18next.t<string>('technicalAssistance:table.header.requestName'),
+  i18next.t<string>('technicalAssistance:adminHome.requester'),
+  i18next.t<string>('technicalAssistance:adminHome.requestType'),
+  i18next.t<string>('technicalAssistance:adminHome.trbLead'),
+  i18next.t<string>('technicalAssistance:adminHome.status'),
+  i18next.t<string>('technicalAssistance:table.header.trbConsultDate')
+] as const;
+
+export function getTrbRequestDataAsCsv(requests: TrbAdminTeamHomeRequest[]) {
+  const rows = requests.map(r => {
+    const submissionDate = r.form.submittedAt
+      ? formatDateLocal(r.form.submittedAt, 'MM/dd/yyyy')
+      : '';
+    const trbConsultDate = r.consultMeetingTime
+      ? formatDateLocal(r.consultMeetingTime, 'MM/dd/yyyy')
+      : '';
+    const requester = getPersonVal(
+      r.requesterInfo.commonName,
+      r.requesterComponent
+    );
+    const trbLead = getPersonVal(r.trbLeadInfo.commonName, r.trbLeadComponent);
+
+    return [
+      submissionDate,
+      r.name,
+      requester,
+      i18next.t<string>(`technicalAssistance:table.requestTypes.${r.type}`),
+      trbLead,
+      i18next.t<string>(`technicalAssistance:table.requestStatus.${r.status}`),
+      trbConsultDate
+    ];
+  });
+
+  return [trbRequestsCsvHeader, ...rows];
+}
+
+function CsvDownloadLink({
```

Comment:
Just a thought, but it might be worth moving this to the shared components folder since you're already breaking it out into a separate component.

Reply:
Ok I'll go for it